src/NodeOperatorManager.sol

```diff
@@ -36,11 +36,17 @@ contract NodeOperatorManager is INodeOperatorManager, IPausable, Initializable,
     mapping(address => bool) private whitelistedAddresses;
     mapping(address => bool) public registered;
 
-    mapping(address => bool) public admins;
+    mapping(address => bool) public DEPRECATED_admins;
     mapping(address => mapping(ILiquidityPool.SourceOfFunds => bool)) public operatorApprovedTags;
 
     RoleRegistry public roleRegistry;
 
+    //--------------------------------------------------------------------------------------
+    //-------------------------------------  ROLES  ---------------------------------------
+    //--------------------------------------------------------------------------------------
+
+    bytes32 public constant NODE_OPERATOR_ADMIN_ROLE = keccak256("NODE_OPERATOR_ADMIN_ROLE");
```

Comment:
Purely for consistency lets have `NODE_OPERATOR_MANAGER_ADMIN_ROLE`

Reply:
fixed